nftables-zoned.nix

```diff
@@ -65,6 +65,16 @@ with dependencyDagOfSubmodule.lib.bake lib;
             type = listOf int;
             default = [];
           };
+          allowedTCPPortRanges = mkOption {
+            type = listOf (attrsOf port);
```

Comment:
If the only allowed keys in the attrset are "from" and "to", I think `listOf (submodule ...)` should be used instead. Would you mind implementing that?